src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.fileReader;
+import core.basesyntax.dao.fileReaderImpl;
+import core.basesyntax.dao.writeReportImpl;
```

Comment:
The class name `writeReportImpl` does not follow Java naming conventions. Class names should be in PascalCase, such as `WriteReportImpl`.